src/acl.js

```diff
@@ -254,6 +276,24 @@ util.inherits(Acl, AclRoleAccessorMethods);
  *   var aclObject = data[0];
  *   var apiResponse = data[1];
  * });
+ *
+ * @example <caption>include:storage/acl.js</caption>
+ * region_tag:storage_add_file_owner
+ * organization:GoogleCloudPlatform
+ * repository:nodejs-docs-samples
+ * Example of adding an owner to a file:
+ *
+ * @example <caption>include:storage/acl.js</caption>
```

Comment:
Seems like these should be:

```
@example <caption>include:samples/acl.js</caption>
region_tag:storage_add_file_owner
organization:googleapis
repository:nodejs-storage
```

Thoughts?

P. S. Are `organization` and `repository` necessary if they point to the same repo?

Reply:
Sent you a response

Reply:
The need for those was temporary, I've removed them